scrape_repo_information.py

```diff
@@ -0,0 +1,88 @@
+import urllib.request
+import pdb
+import xml.etree.ElementTree as xml
+import argparse, urllib.request
+import json
+import collections
+import time
+import urllib
+import os
+"""
```

Comment:
Please add newlines around the docstring and include an example of what an entry in the output file looks like.